File: 30 days of code( python)/day8.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
n = int(input())
d = {}
while n>0 :
    s = input().strip().split()
    d[s[0]] = s[1]
    n = n-1

while True:
    try:
        temp = input()
        if temp in d.keys():
            print( temp+"="+d[temp] )  
        else:
            print("Not found")
    except:
        break

File: 30 days of code( c++)/day8.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <map>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int n;
string name;
long num;
cin >> n;
cin.ignore();
map <string, long> pBook;
for (int i = 0; i < n; i++) {
    cin >> name;
    cin >> num;
    pBook[name] = num;
}
while(cin >> name) {
    if (pBook.find(name) != pBook.end()) {
        cout << name << "=" << pBook.find(name)->second << endl;
    } else {
        cout << "Not found" << endl;
    }
}
    return 0;
}

